psalm-baseline.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Surprisingly small size of baseline 🤔

Reply:
Most of the issues were in tests. https://github.com/laminas/laminas-mvc-plugin-flashmessenger/pull/21/files#r780972781 might squash several more if I can figure it out.

Reply:
I think it's fine as-is as a start